CONTRIBUTING.md

```diff
@@ -19,6 +19,21 @@ If you want to make a bigger change, it's a good idea to first file an issue and
 - Make your changes, commit to git, and then create a PR by running `usethis::pr_push()`, and following the prompts in your browser. The title of your PR should briefly describe the change. The body of your PR should contain Fixes #issue-number.
 - For user-facing changes, add a bullet to the top of `NEWS.md` (i.e. just below the first header). Follow the style described in https://style.tidyverse.org/news.html.
 
+### What happens after submitting a pull request?
+
+PRs are reviewed by the team before they are merged. The review process begins after continuous integration checks, which can take some time. We do not expect contributors to familiarise themselves with all the automated checks in our GitHub actions, and are happy to support on this once a PR is made. 
+
+However, for those who are interested in running local checks to accelerate the PR process, here is a summary of the main checks that are automatically performed and how they can be confirmed:
```

Comment:
So, I understand this to mean that the local checks are optional but I wouldn't suggest that. I would rather suggest that they make sure the local checks are passing and we can help them with the CI check failures. If users are struggling with the local checks, they can always post their log in the issue thread for help. If we don't enforce that, it will become quite a burden (especially in phase 2, where there will be only one RSE) to always fix CI failures that could have been prevented with local checks, and also considering that for large packages, these take some time.

@Bisaloo and @chartgerink thoughts?

Reply:
What about "we recommend you run local checks" as a middle ground between making it sound like unnecessary and making it mandatory?

> If we don't enforce that, it will become quite a burden (especially in phase 2, where there will be only one RSE) to always fix CI failures that could have been prevented with local checks

I expect contributors who are able to fix local test failures to be able to fix CI test failures as well. As such, I think the maintainer work is more or less the same in both cases.